libgalois/src/analytics/louvain_clustering/louvain_clustering.cpp

```diff
@@ -380,13 +380,17 @@ struct LouvainClusteringImplementation
         !result) {
       return result.error();
     }
-    std::vector<std::string> temp_edge_property_names = {
-        "_katana_temporary_property_" + edge_weight_property_name};
     if (auto result = ConstructEdgeProperties<EdgeData>(
             pfg_mutable.get(), temp_edge_property_names);
         !result) {
       return result.error();
     }
+    */
+
+    std::vector<std::string> temp_edge_property_names = {
```

Comment:
There is: https://github.com/KatanaGraph/katana/blob/master/libgalois/include/katana/analytics/Utils.h#L75

Reply:
Done.